DEVELOPMENT.md

```diff
@@ -93,25 +110,30 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible for query execution and managing PEMs.
 
-Deploying a development version of Vizier is a 2-step process. A release-version of Vizier must first be deployed (through the CLI or YAMLs), which will set up specific cluster-secrets that are not deployed via Skaffold. Skaffold can then be run to build and deploy a version of Vizier based on local changes.
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
 
-1. If deploying Vizier pointing to a dev cloud instance, make sure to export the following environment variables:
+The following will build Pixie Vizier and run a unit test of your choice in Pixie's Vizier module. If you are running the development environment via Docker, the following should be run inside the Docker container.
 
-    ```
-    export PL_CLOUD_ADDR=dev.withpixie.dev:443
-    export PL_TESTING_ENV=dev
-    ```
+```
+bazel test //src/<path to unit test file> --test_output=errors -j $(nproc)
+```
+
+### Deploying
+Deploying a development version of Vizier is a 2-step process. An official release-version of Vizier must first be deployed (through the Pixie CLI or YAMLs) and Skaffold can then be run to build and deploy a local development version of Vizier.
 
-2. Deploy Vizier through CLI/YAML. If deploying Vizier to a dev cloud instance, make sure to export the necessary dev-specific environment variables mentioned in `Pixie Cloud` above.
+1. If you wish to test development changes made to both Pixie Cloud and Vizier, export the following environment variables that will point to the development Pixie Cloud instance:
 
     ```
-    px deploy
+    export PL_CLOUD_ADDR=dev.withpixie.dev:443
+    export PL_TESTING_ENV=dev
     ```
 
-3. Wait for the Vizier to successfully connect to the cloud instance, to ensure all secrets and configs have been set up properly.
+2. Install the Pixie CLI and run `px deploy`. Depending on whether you are pointing to a self-hosted Pixie Cloud or the official Community Pixie Cloud follow the appropriate installation guide [here](https://docs.px.dev/installing-pixie/install-guides/). `px deploy` will set up specific cluster-secrets, etc that are not deployed via Skaffold. Wait for this command to successfully complete and Vizier to successfully connect to Pixie Cloud, to ensure all secrets and configs have been set up properly. Note you will not need to run this to deploy again unless you connect to a different cluster.
 
-4. Deploy a local version of Vizier using skaffold.
+3. Deploy a local development version of Pixie Vizier using Skaffold. Note each time you make a code change you will need to run this command to build and deploy the new version.
```

Comment:
Minor but we might want to link to skaffold's docs like [Skaffold](https://skaffold.dev/) and same with bazel